macOS/UITests/DuckPlayerTests.swift

```diff
@@ -383,20 +386,4 @@ class DuckPlayerTests: UITestCase {
         XCTAssertFalse(watchLink.waitForExistence(timeout: 1))
     }
 
-    func test_DuckPlayer_AskMode_OpensYouTube_WithOverlay() throws {
-        // Settings
-        openDuckPlayerSettings()
-        selectAskOpenInDuckPlayer()
-        app.closeCurrentTab()
-
-        // Search
-        openURL(url: Self.youtubeURLForVideo)
-
-        verifyYoutubeLoads()
-
-        // Overlay visible
-        let watchLink = app.links.containing(.staticText, identifier: Self.turnOnDuckPlayer).firstMatch
-        XCTAssertTrue(watchLink.waitForExistence(timeout: 1))
-    }
```

Comment:
I caught up with Omid, who said this test can be removed.

Reply:
I don't think it can be removed, but let's do for now while I investigate a fix